cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
```

Comment:
The `validate_ticket` method is being called on the `Ticket` model, but it's not clear if this method exists and is implemented correctly. Ensure that `validate_ticket` is a valid method of the `Ticket` model and that it accepts the parameters being passed to it. Additionally, ensure that `ValidationError` is being used correctly within this method.